web/packages/teleport/src/Integrations/status/AwsOidc/Tasks/Tasks.tsx

```diff
@@ -95,6 +99,15 @@ export function Tasks() {
       // If there are multiple pages, we would rather refresh the table with X results rather than
       // use modifyFetchedData to remove the item.
       serverSidePagination.fetch();
+
+      setNotification({
+        content: {
+          description:
+            'The task has been marked as resolved; it will reappear in the table if the issue persists in the next sync.',
```

Comment:
```suggestion
            'The task has been marked as resolved; it will reappear in the table if the issue persists after the next sync.',
```
maybe? you pick